go/azidext/go.mod

```diff
@@ -0,0 +1,14 @@
+module azidext
```

Comment:
the module identity should be fully qualified

`github.com/jongio/azidext/go/azidext`